src/utils/power_flow/power_flow.jl

```diff
@@ -146,33 +133,16 @@ Arguments available for `nlsolve`:
 ## Examples
 ```julia
 using NLsolve
-@solve_powerflow!(sys)
+solve_powerflow!(sys, nlsolve)
 # Passing NLsolve arguments
-@rsolve_powerflow!(sys, method = :Newton)
+solve_powerflow!(sys, nlsolve, method = :Newton)
 
 ```
 
 """
-
-macro solve_powerflow!(sys, args...)
-    vals = filter((x) ->  x == :NLsolve, names(Main,imported=true))
```

Comment:
I think we should keep these two lines, makes for a better user experience and it is not a super problematic thing to run